seed/models/analysis_property_views.py

```diff
@@ -95,19 +95,19 @@ def get_property_views(cls, analysis_property_views):
         property_view_query = models.Q()
         for analysis_property_view in analysis_property_views:
             property_view_query |= (
-                models.Q(property=analysis_property_view.property)
-                & models.Q(cycle=analysis_property_view.cycle)
+                models.Q(property_id=analysis_property_view.property_id)
```

Comment:
I'm pretty sure this is functionally the same, but being explicit that it is the `_id` is nicer.

Reply:
I could be wrong, but I think before it was retrieving the whole cycle and property from the db, which cost us an extra query per loop.  

I remember less queries being logged after the change, but there were a lot of logs, so I could be confused.